packages/docusaurus-plugin-content-docs/src/__tests__/lastUpdate.test.ts

```diff
@@ -7,9 +7,10 @@
 
 import fs from 'fs';
 import path from 'path';
-import shell from 'shelljs';
+import spawn from 'cross-spawn';
```

Comment:
Do we even need to import these? I had to import shelljs because I was mocking it

Reply:
```suggestion
```